x-pack/test/fleet_functional/apps/fleet/overview_page.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const { overviewPage } = getPageObjects(['overviewPage']);
+
+  describe('When in the Fleet application', function () {
+    this.tags(['ciGroup7']);
```

Comment:
ping @elastic/kibana-operations for CI group balancing

`test/fleet_api_integration/*` is `ciGroup10` but I doubt that matter